docs/useNullableState.md

```diff
@@ -0,0 +1,51 @@
+# OneBlink Apps-react | useNullableState Hook
+
+## `useNullableState()`
+
+This function is a react hook for state of type of your choosing. It comes with a `useCallback` for setting the state.
+
+- Note: `T` represents the type of the state you pass as a default value.
+
+## Paramaters
+
+| Property       | Type | Description                      |
+| -------------- | ---- | -------------------------------- |
+| `defaultValue` | `T`  | The starting state for the hook. |
+
+## Return
+
+The return type of `useNullableState(myState)` is an array where:
+
+- The first item is of type `T` (the state).
+- The second item is `(newValue: T) => void` (a function that sets the state to the `newValue`).
```

Comment:
There is a third element in the array which is a function to clear the state (set to `null`)

Reply:
ahh, so there is.